### PR TITLE
[wip] fantasticon post

### PR DESCRIPTION
**Pull request by hougasian**

### CODE REVIEW

File: _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+| source | default |
+|---|---|
+| artboard | 4" x 4" |
+| main stroke | 24pt |
+| auxiliary stroke | 18pt |
+| corner | .15" |
+| join |	round |
+
+
+There are a few tools out there you can evaluate for your workflow. We currently use SVGO, but will incorporate outlie-stroke in a future build.
```
Comment (mrtrimble) [inline _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md:31]: Typo: `outlie-stroke` -> `outline-stroke`

File: _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+```js
+module.exports = {
+  // We run the main icon directory through SVGO first to 
+  // output the inputDir.  
+  inputDir: './src/optimized-icons',
+  // Where Vite puts things
+  outputDir: './public',
+  // What font types to generate; If you're soley generating 
+  // for the we, you only need woff and woff2
```
Comment (mrtrimble) [inline _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md:66]: Typo: `For the we` -> `For the web`

File: _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+twitter_text: "Generating your own fonts with Fantasicon"
+authors: Kevin Hougasian
+image: https://images.unsplash.com/photo-1609605348579-3123e3d40eb8?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1674&q=80
+image_url: https://unsplash.com/photos/92-mTYj5oGs
+image_credit: Brett Jordan
+---
+
+Anyone who's heard me mention [Platform UI](https://platformui.com/) probably knows we make our own font icons, and bake them right into Platform UI, to keep it a one-stop resource. There are some wonderful font packages out there, but you don't always want to load thousands of icons to get the particular mix you're going for. 
+
+For anyone interested, here what I found along the way...
```
Comment (tedk13) [inline _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md:16]: Here's?
```suggestion
For anyone interested, here is what I found along the way...
```

**Review by tedk13 — CHANGES_REQUESTED** [review]

File: _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+  assetTypes: ['scss'],
+  prefix: 'pi',
+  name: "platform-icons"
+};
+``` 
+Fontasticon has a great [Readme](https://github.com/tancredi/fantasticon#readme) and [Discord channel](https://discord.gg/BXAY3Kc3mp). I couldn't have done this without BOTH of them! 
+
+A big issue we faced was adding new icons - we would overwrite previous values making the use of `:before` and `:after` impossible. Enter `codepoints`; But how did they really work? 
+
+Let walk through our current config file `.fantasticonrc.js` to see how it all comes together. We use [Vite](https://vitejs.dev/) to generate Platform UI (with icons) and a stand-alone Platform-icons repo.
```
Comment (tedk13) [inline _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md:56]:
```suggestion
Let's walk through our current config file `.fantasticonrc.js` to see how it all comes together. We use [Vite](https://vitejs.dev/) to generate Platform UI (with icons) and a stand-alone Platform-icons repo.
```

**Review by tedk13 — CHANGES_REQUESTED** [review]

File: _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md (hunk 4 above)
Comment (hougasian) [reply _posts/2023-02-27-generating-your-own-fonts-with-fantasicon.md:56]: yeah, I did not run this through Grammarly :(

**Review by tedk13 — APPROVED** [review]

**Comment by mrtrimble** [comment]
Added this post to #490 can probably close this.

**Pull request closed without merging**